benefits/core/templates/core/includes/form--select-agency.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+
+<form class="{{ form.classes }}" id="{{ form.id }}" method="{{ form.method | default:"post" | upper }}">
+  {{ form.as_p }}
```

Comment:
Doesn't feel right to wrap this field in a `<p>`. Removing this will revert it to just being a `<div>`:


```suggestion
  {{ form }}
```

This does require a little additional styling, which I'll suggest in a later comment.